content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst

```diff
@@ -142,13 +140,12 @@ Process a delivery order in two steps (pick + ship)
 Create a sales order
 --------------------
 
-In the :menuselection:`Sales` application, create a new quote by clicking :guilabel:`Create`. Select
-or create a :guilabel:`Customer`, add a storable :guilabel:`Product` to the order lines, and then
+In the :menuselection:`Sales` application, create a new quote by clicking :guilabel:`New`. Select
+(or create) a :guilabel:`Customer`, add a storable :guilabel:`Product` to the order lines, and then
 click :guilabel:`Confirm`.
 
-After confirming the :abbr:`SO (Sales Order)`, a :guilabel:`Delivery` smart button will appear in
-the top-right corner of the :abbr:`SO (Sales Order)` form. Click it to reveal the associated
-delivery forms.
+After confirming the :abbr:`SO (Sales Order)`, a :guilabel:`Receipt` smart button will appear in
+the top, above the :abbr:`SO (Sales Order)` form — click it to reveal the associated receipt.
```

Comment:
```suggestion
the top, above the :abbr:`SO (Sales Order)` form—click it to reveal the associated receipt.
```

Reply:
I realized after doing the images for the new Milk design- it is a DELIVERY smart button, not a receipt! I changed it so it is correct.